core/js/typespecs/prompt.ts

```diff
@@ -80,6 +80,15 @@ const openAIModelParamsSchema = z
           .strip(),
       ])
       .optional(),
+    function_call: z.union([
+      z.literal("auto"),
+      z.literal("none"),
+      z.strictObject({
```

Comment:
Should we add `.strip()` here if we are unsure whether there might be more properties than `name`?

Reply:
I think it's better to keep it strict, especially since this is a deprecated parameter.